### PR TITLE
koord-scheduler: support default preferredCPUBindPolicy for LSE/LSR P…

### PR DESCRIPTION
**Pull request by eahydra**
…od if not specified

Signed-off-by: Joseph <joseph.t.lee@outlook.com>

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: pkg/scheduler/plugins/nodenumaresource/plugin.go [hunk 1]
```diff
@@ -132,8 +133,12 @@ func (p *Plugin) PreFilter(ctx context.Context, cycleState *framework.CycleState
 	qosClass := extension.GetPodQoSClass(pod)
 	priorityClass := extension.GetPriorityClass(pod)
 	if (qosClass == extension.QoSLSE || qosClass == extension.QoSLSR) && priorityClass == extension.PriorityProd {
-		if resourceSpec.PreferredCPUBindPolicy == schedulingconfig.CPUBindPolicyFullPCPUs ||
-			resourceSpec.PreferredCPUBindPolicy == schedulingconfig.CPUBindPolicySpreadByPCPUs {
+		preferredCPUBindPolicy := resourceSpec.PreferredCPUBindPolicy
+		if preferredCPUBindPolicy == "" || preferredCPUBindPolicy == schedulingconfig.CPUBindPolicyNone {
```
Comment (saintube) [inline pkg/scheduler/plugins/nodenumaresource/plugin.go:137]: is it possible to specify bind no CPUs?

Comment (eahydra) [reply pkg/scheduler/plugins/nodenumaresource/plugin.go:137]: LSE/LSE Pods must specify CPUBindPolicy, which cannot be set to None.

Comment (saintube) [reply pkg/scheduler/plugins/nodenumaresource/plugin.go:137]: can we remove the `CPUBindPolicyNone` option

Comment (eahydra) [reply pkg/scheduler/plugins/nodenumaresource/plugin.go:137]: hmm..It's better to rename `CPUBindPolicyNone` to `CPUBindPolicyDefault`. WDYT?

Comment (saintube) [reply pkg/scheduler/plugins/nodenumaresource/plugin.go:137]: totally agree

**Pull request merged**